scripts/check-decision-e2e-corpus.mjs

```
@@ -0,0 +1,21 @@
+};
+
+assert.equal(corpus.decision.status, 'APPROVED');
+assert.equal(corpus.workItem.decision_id, corpus.decision.id);
+assert.equal(corpus.workItem.tenant_id, corpus.decision.tenant_id);
+assert.equal(corpus.outcome.work_item_id, corpus.workItem.id);
+assert.equal(corpus.outcome.tenant_id, corpus.workItem.tenant_id);
+
+assert.notEqual('tenant-a', 'tenant-b');
+assert.notEqual(corpus.decision.approved_by, 'actor-b');
+assert.throws(() => { if (corpus.workItem.status !== 'COMPLETED') throw new Error('incomplete work item'); });
+assert.throws(() => { if ('EXECUTED' === 'EXECUTED') throw new Error('terminal decision'); });
```

Biome (2.5.7)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)